disnake/client.py

```diff
@@ -285,6 +285,15 @@ class Client:
 
         .. versionadded:: 2.0
 
+    enable_gateway_error_handler: :class:`bool`
+        Whether to enable the :func:`disnake.on_gateway_error` event.
+
+        If this is disabled, exceptions that occur while parsing (known) gateway events
+        won't be handled and the pre-v2.6 behavior of letting the exception propagate up to
+        the :func:`connect`/:func:`start`/:func:`run` call is used instead.
+
+        .. versionadded:: 2.6
+
```

Comment:
what does this default to? should document that here.

Reply:
Added in https://github.com/DisnakeDev/disnake/pull/401/commits/50460bc774c1727aea6e5febf80ddd28df11316b, the actual default value is still up in the air somewhat